### ISSUE TITLE
NameObfuscation access bug

### ISSUE THREAD

**Issue by superblaubeere27** [superblaubeere27/obfuscator#50]
# Bug Report
### Expected Behaviour
The Name Obfuscation would leave the methods' access of excluded classes untouched
### Actual Behaviour
It changes everything to public
### How to reproduce the behaviour
- Try to obfuscate a jar with some excluded classes which contains private methods
### Possible fix
Make a check if the class is not remapped

**Comment by cookiedragon234** [comment]
Example:
![Before Obfuscation](https://media.discordapp.net/attachments/617022892017975296/617455275598807050/unknown.png)
![After Obfuscation](https://cdn.discordapp.com/attachments/617022892017975296/617455339834441757/unknown.png)

Causes crash
```
[20:45:39] [main/FATAL] [mixin]: Mixin apply failed mixins.backdoored.json:MixinLayerArmorBase -> net.minecraft.client.renderer.entity.layers.LayerArmorBase: org.spongepowered.asm.mixin.transformer.throwables.InvalidMixinException Mixin mixins.backdoored.json:MixinLayerArmorBase contains non-private static method renderEnchantedGlint(FFFF)V
```

**Comment by superblaubeere27** [comment]
Fixed in e41b971ae782d62eabe273f4b4af37309e9f755b
